Examples/ScaleMicrogrid/SolutionKeys.hpp

```diff
@@ -4,8 +4,11 @@
  * @author Slaven Peles (peless@ornl.gov)
  * @brief Answer keys for Scaled Microgrid test with Nsize = 2, 4, 8
  * 
- * All data generated with Matlab ode23tb solver with tolerances set to
- * abstol=1e-12 and reltol=1e-12.
+ * Data generated with Matlab ode23tb solver with tolerances set to
+ * abstol = 1e-12 and reltol = 1e-12 for the ODE derivation.
+ * 
+ * DAE keys are generated with Matlab IDA interface with tolerances set to
+ * abstol = 1e-10 and reltol = 1e-10 for the DAE derivation.
```

Comment:
Which one of the two is true?

Reply:
There is only one set of data with reference results. Comments in the code should describe only the data that is included.

Reply:
??? There are two set of keys. ODE keys are denoted `answer_key_N*` and DAE keys are denoted `answer_key_N*_DAE`. Both reference data sets are in the solution keys file.